presets/input-history-windows/input-history-windows.html

```diff
@@ -44,9 +44,12 @@
         transition-duration: 1s;
       }
 
-      p.key-combination > * {
+      p.key-combination {
         /* For line-height */
         margin-top: 0.5em;
+
+      p.key-combination > * {
+        vertical-align: middle;
       }
```

Comment:
These changes seem to have introduced the removal of the background as well as the size issues, commenting them out gives me this:
![image](https://user-images.githubusercontent.com/8353672/230875058-5e910ee1-2056-4b49-8e63-804f18b6b321.png)
The mouse wheel up/down aren't perfectly aligned but I'd say that's fine